cmd/bug-automation/generate/main.go

```diff
@@ -258,6 +280,13 @@ func doGenerate() error {
 			Update:      bugsWithUpcomingSprintUpdate(),
 			Default:     false,
 		},
+		{
+			Name:        "removeReviewedInSprint",
+			Description: "Set reviewed-in-sprint flag to '-' in all bugs",
+			Query:       bugsWithReviewedInSprintFlagQuery(),
+			Update:      bugsWithReviewedInSprintFlagUpdate(),
+			Default:     false,
+		},
```

Comment:
i made this a new command rather than adding logic to the existing command because it was easier given the structure of things.  I figure for whatever duration we have overlap, we can just run both remove commands, unless you think that's too expensive.

Reply:
Can you update the Makefile to run both?

Reply:
oh, you need me to compile, harumph.

Reply:
I created https://github.com/openshift/bugzilla-tools/pull/33   after we merge that we need to run the generator, update the Makefile to run both and merge this.

Reply:
i rebased on your PR, updated the makefile, and ran the generator and added the new yaml file.